Problem: Can not connect to the streaming server

After making clean install from clone git I can't connect to icecast server. 
![screen shot 2017-03-20 at 11 02 10](https://cloud.githubusercontent.com/assets/25418594/24092039/e7804fa0-0d5c-11e7-8048-6f0874464903.png)
If i choose 

> Custom / 3rd Party Streaming
![screen shot 2017-03-20 at 11 07 47](https://cloud.githubusercontent.com/assets/25418594/24092144/7c263ba6-0d5d-11e7-9821-474a89c56c62.png)

If I install from tarball it working fine.
liquidsoap logs are empty, icecast shows no errors too. Please tell me where/how to search the bug.

```
$ sudo su - www-data -s /bin/bash -c 'python /usr/bin/airtime-liquidsoap'
Airtime Liquidsoap
Unable to connect to the Airtime server.
<urlopen error [Errno 1] _ssl.c:510: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol>
traceback: Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/airtime_playout-1.0-py2.7.egg/liquidsoap/generate_liquidsoap_cfg.py", line 48, in run
    ss = ac.get_stream_setting()
  File "/usr/local/lib/python2.7/dist-packages/api_clients-1.0-py2.7.egg/api_clients/api_client.py", line 432, in get_stream_setting
    return self.services.get_stream_setting()
  File "/usr/local/lib/python2.7/dist-packages/api_clients-1.0-py2.7.egg/api_clients/api_client.py", line 144, in __call__
    f = urllib2.urlopen(req, timeout=ApiRequest.API_HTTP_REQUEST_TIMEOUT)
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/usr/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 1222, in https_open
    return self.do_open(httplib.HTTPSConnection, req)
  File "/usr/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: <urlopen error [Errno 1] _ssl.c:510: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol>


```

--- a/python_apps/api_clients/api_clients/api_client.py
+++ b/python_apps/api_clients/api_clients/api_client.py
@@ -187,7 +187,7 @@ class RequestProvider(object):
         if self.config["general"]["base_dir"].startswith("/"):
             self.config["general"]["base_dir"] = self.config["general"]["base_dir"][1:]
         self.url = ApcUrl("%s://%s:%s/%s%s/%s" \
-            % (str(("https", "http")[self.config["general"]["base_port"] == 443]),
+            % (str(("http", "https")[int(self.config["general"]["base_port"]) == 443]),
                self.config["general"]["base_url"], str(self.config["general"]["base_port"]),
                self.config["general"]["base_dir"], self.config["api_base"],
                '%%action%%'))
@@ -325,7 +325,7 @@ class AirtimeApiClient(object):
         if self.config["general"]["base_dir"].startswith("/"):
             self.config["general"]["base_dir"] = self.config["general"]["base_dir"][1:]
         url = "%s://%s:%s/%s%s/%s" %  \
-            (str(("https", "http")[self.config["general"]["base_port"] == 443]),
+            (str(("http", "https")[int(self.config["general"]["base_port"]) == 443]),
              self.config["general"]["base_url"], str(self.config["general"]["base_port"]),
              self.config["general"]["base_dir"], self.config["api_base"],
              self.config[config_action_key])

